jumpscale/threesdk/container.py

```diff
@@ -43,6 +43,7 @@ def start(name):
         if not docker_client.exists(name):
             raise Value("Container with specified name doesn't exist")
         docker_client.start(name)
+        docker_client.exec(name, ["threebot, start"])
```

Comment:
shouldnt be as part of the container class
but only in the threebot class which overwrites this method (same for stop)

Reply:
also this should be ["threebot", "start"]